macros/livemode_predicate.sql

```diff
@@ -1,5 +1,5 @@
 {% macro livemode_predicate() %}
 
-    where livemode = {{ var('stripe__using_livemode', true) }}
+    where cast(livemode as boolean) = {{ var('stripe__using_livemode', true) }}
```

Comment:
Sorry to miss this - yeah, the cast was there because integration tests was failing because I think one of the models was not reading livemode as a boolean and I wanted this where clause to work. Also I didn't realize there was a dbt macro for it because it was passing for all so assumed it was cross-db friendly, but you're right!